guides/common/modules/proc_creating-hosts-from-discovered-hosts.adoc

```diff
@@ -1,5 +1,5 @@
 [id="Creating_Hosts_from_Discovered_Hosts_{context}"]
-= Creating Hosts from Discovered Hosts
+= Creating hosts from discovered hosts
```

Comment:
```suggestion
= Creating hosts from Discovered hosts
```

Reply:
Not sure about this. "discovered" and "discovery" are lowercase in the text. I think it's OK for them to be lower case in this context. I would capitalize "Discovery service" but not "discovered hosts" because this use is generic.

Reply:
I am OK either way; this minor detail should not block your PR.